srcpkgs/python3-tasklib/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-tasklib'
+pkgname=python3-tasklib
+version=2.4.3
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-wheel python3-pip task"
```

Comment:
```suggestion
hostmakedepends="python3-setuptools task"
```

Reply:
Thanks, for catching that

Reply:
Hmm, the checks fail without those. Are you sure we don't need them?

Reply:
put them to checkdepends and also add pytest

Reply:
I meant to add the dependencies there, not pip

Reply:
All host dependencies?